hybrid/ddc_null.c

```diff
@@ -0,0 +1,20 @@
+/***
+ * This programs shows that putting
+ * "NULL" into the DDC causes an
```

Comment:
Same comment as above: where is the exception caused?

Reply:
Assertions in 7548354.